core/banking/gov_transfers.go

```diff
@@ -0,0 +1,271 @@
+package banking
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"code.vegaprotocol.io/vega/core/events"
+	"code.vegaprotocol.io/vega/core/types"
+	"code.vegaprotocol.io/vega/libs/num"
+	"code.vegaprotocol.io/vega/logging"
+	vegapb "code.vegaprotocol.io/vega/protos/vega"
+)
+
+func (e *Engine) distributeScheduledGovernanceTransfers(ctx context.Context) {
+	timepoints := []int64{}
+	now := e.timeService.GetTimeNow()
+	for k := range e.scheduledGovernanceTransfers {
+		if now.UnixNano() >= k {
+			timepoints = append(timepoints, k)
+		}
+	}
+
+	for _, t := range timepoints {
+		transfers := e.scheduledGovernanceTransfers[t]
+		for _, gTransfer := range transfers {
+			amt, err := e.processGovernanceTransfer(ctx, gTransfer)
+			if err != nil {
+				gTransfer.Status = types.TransferStatusStopped
+				e.broker.Send(events.NewGovTransferFundsEventWithReason(ctx, gTransfer, amt, err.Error()))
+			}
```

Comment:
Should we be sending an event with the status set to `types.TransferStatusDone` when theres not an error? I can't see anywhere we do this other than when the one-off-gov-transfer happens right away.

Reply:
sure.